### DISCUSSION TITLE
ＡＩ也會種族歧視

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#411] [General]
原本以為人類才會歧視，電腦應該是最公平的，但沒想到電腦也默默地提取人類沒教的資訊，包括人種及性別等，因此如何去除歧視，讓各族群的醫學影像判讀都有一致的準確率，會成為下一階段要努力的目標。

https://technews.tw/2022/06/09/medical-ethics-concerns/

**Comment by luo910227** [comment]
我覺得這絕對是研發者需要非常認真關注的問題，人類的互相歧視已經停不下來了完全容不下連機器人都會歧視。

**Comment by lx3211** [comment]
還是其實是資料集的數量不同造成誤判率的差異？

**Comment by brian431** [comment]
AI確實經常在不知不覺中產生與人類相同的歧視行為，而現今的人工智慧開發者大部分都有注意到這個問題了，在之後應該會有很大的改進。如何讓社會規範跟上科技的發展是一個重要的議題。

**Comment by xCk27x** [comment]
我覺得這不像是所謂AI歧視耶
比較像是人體構造差異帶來的誤判率

**Comment by NcuJerryKao** [comment]
還蠻吃驚的 不知道AI還會不知不覺訓練出什麼 也許有不明顯的 人類尚未察覺
例如負面的 邪惡 欺騙 反人類思維

**Comment by Ben6uo** [comment]
那是跟訓練的內容有關吧

**Comment by awkward-willy** [comment]
前幾年 Amazon 原想以機器學習開發自動化批准履歷的系統，但因為女性在訓練的樣本數不足導致無論如何都有性別上的差別對待，最終失敗告終。所以如何消弭歧視或許是未來重要的課題之一。

**Comment by BingRong1217** [comment]
是個蠻重要的議題，這個社會不應該存在歧視

**Comment by 32844583** [comment]
在DELL-E(文字產生圖片的AI)也有這種現象，輸入 CEO 大多都只出現白人

**Comment by limitx0** [comment]
黑人和白人誤判率不同單純是模型的問題吧
不過能從X光片判斷人種真的很神奇

**Comment by skyling0299** [comment]
美國警務系統舉例人類在 positive design intent 的實例，當地警察對特定人種犯罪已有特定偏見，當 AI 提供的資訊較傾向該人種時，儘管數據差異不大，警察仍然會認為 AI「認同」他的想法，進而排除其他可能性。

**Comment by Yilin20021016** [comment]
感覺是因為大多數人只有去歧視 而比較小去做平反所令資料分析時出現偏差?

**Comment by boson13579** [comment]
AI是根據給他的資料進行分析、整理、統整再回饋，所以會歧視的應該不是AI，而是人類。是人類發了與種族歧視相關的文章，被AI讀入之後才被誤解為AI也會歧視。
